mongo_connector/connector.py

```diff
@@ -1103,7 +1104,7 @@ def setup_logging(conf):
             interval=conf['logging.rotationInterval'],
             backupCount=conf['logging.rotationBackups']
         )
-        print("Logging to %s." % conf['logging.filename'])
+        print("Logging to %s." % os.path.abspath(conf['logging.filename']))
```

Comment:
Is this consistently converted to an absolute path in other places? Perhaps this should be converted to an absolute path when parsing the config file/CLI options?